packages/2d/src/components/Icon.ts

```diff
@@ -0,0 +1,89 @@
+import {Image, ImageProps} from '../components/Image';
+import {colorSignal, computed, initial, signal} from '../decorators';
+import {SignalValue, SimpleSignal} from '@motion-canvas/core/lib/signals';
+import {useLogger} from '@motion-canvas/core/lib/utils';
+import {ColorSignal} from '@motion-canvas/core/lib/types';
+
+export interface IconProps extends ImageProps {
+  /**
+   * Expects an Icon Identifier.
+   * You can find identifiers on [Icônes](https://icones.js.org/collection/all).
+   * They can look like this:
+   * * mdi:language-typescript
+   * * ph:anchor-simple-bold
+   * * ph:activity-bold
+   */
```

Comment:
The properties from the interfaces should inherit the documentation from the class:
```suggestion
  /**
   * {@inheritDoc Icon.icon}
   */
```